### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by veso266** [SopaXorzTaker/palmod#1]
Hi may I ask what PAL System this modulator will output for (B,G, I,L,M,N, D,K?)

and what is the best way to convert yuv video into this raw format?

and is it posible to pipe teletext data in?

Thanks for anwsering and Best Regards

**Comment by SopaXorzTaker** [comment]
It is only in development yet, and **does not work**. Though I am already going to implement the features you were asking for .

**Comment by SopaXorzTaker** [comment]
Though this program is able to modulate a grayscale video signal, and a better version is in development.

**Comment by veso266** [comment]
Hi Thanks for anwsering 

PS: Is this the better version: https://github.com/SopaXorzTaker/analog-tv?

PPS: For Teletext insertion you can look up this: https://github.com/ali1234/raspi-teletext

it can modulate teletext(PAL), and closed captions (NTSC) and you can even feed teletext subtitles in

**Comment by SopaXorzTaker** [comment]
Yes, but it is still in development. You are welcome to contribute though. And huge thanks for linking to the raspi-teletext project :)
